source/includes/_filing_api.md.erb

```diff
@@ -25,6 +25,27 @@ Method  |  `POST`
 URL | `<%= config[:authhost] %>/auth/realms/hmda2/protocol/openid-connect/token`
 Payload  | client_id=hmda2-api <br> grant_type=password <br> username={{username}}%40{{bank_domain}} <br> password={{password}}
 
+### Error Messages
+
+error | error_description | explanation
+---|---|---
+invalid_grant | Account is not fully set up | Some action is required on the user account in order to use it for login. Either the password needs to be reset or the email needs to be verified.
+invalid_grant | Account disabled | Please contact HMDA-Help in order to re-enable the account.
```

Comment:
invalid_grant | Account disabled | Please contact HMDA Help (hmdahelp@cfpb.gov) in order to re-enable the account.